src/test/lro-poller.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as api from "../api";
+import * as FirebaseError from "../error";
+import { LroPoller, LroPollerOptions } from "../lro-poller";
+import TimeoutError from "../throttler/errors/timeout-error";
+import { mockAuth } from "./helpers";
+
+const TEST_ORIGIN = "https://firebasedummy.googleapis.com.com";
+const VERSION = "v1";
+const LRO_RESOURCE_NAME = "operations/cp.3322442424242444";
+
+describe("LroPoller", () => {
+  describe("poll", () => {
+    let sandbox: sinon.SinonSandbox;
+    let stubApiRequest: sinon.SinonStub;
+    let pollerOptions: LroPollerOptions;
+    const poller = new LroPoller();
+
+    beforeEach(() => {
+      sandbox = sinon.createSandbox();
+      mockAuth(sandbox);
+      stubApiRequest = sandbox.stub(api, "request");
+      pollerOptions = {
+        apiOrigin: TEST_ORIGIN,
+        apiVersion: VERSION,
+        operationResourceName: LRO_RESOURCE_NAME,
+        backoff: 10,
+      };
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it("should return result with response field if polling succeeds", async () => {
+      const successfulResponse = {
+        done: true,
+        response: "completed",
+      };
+
+      stubApiRequest
+        .withArgs("GET", `/${VERSION}/${LRO_RESOURCE_NAME}`, {
```

Comment:
Can you extract `/${VERSION}/${LRO_RESOURCE_NAME}` and the auth/origin object into constants for reuse?

Reply:
done.